### ISSUE TITLE
Should consider discount rate?

### ISSUE THREAD

**Issue by XiangLiu-github** [JGCRI/rfasst#86]
Hi,

Thanks for this helpful package! @jonsampedro

I am wondering when I calculate the agriculture revenue losses, should I consider the discount rate?

If I understand the code and description correctly, this package had not taken the discount rate into account. However, in the paper [(Atmospheric Environment)](https://doi.org/10.1016/j.atmosenv.2020.117538), which is referenced in Module 4, the computed net present value is based on a 3% discount rate.

In other words, should I manually use a discount rate to convert the revenue losses to present value? And the price reported by GCAM is not scaled by a discount rate?

**Comment by jonsampedro** [comment]
Hello @XiangLiu-github ,

Yes, as you say, the package calculates undiscounted damages. In order to calculate the NPVs or cumulative damages presented in the paper referenced, you will need to manually apply a discount rate. 

Hope this helps!

**Comment by XiangLiu-github** [comment]
Thanks for your fast response! @jonsampedro 

The thing that confused me is that the unit of price from GCAM output has already been 1975$, is it necessary to use the discount rate to compute NPVs?

**Comment by jonsampedro** [comment]
Yes, that's confusing, but it does not consider a DR, so it needs to be applied manually to compute the NPVs.

**Comment by XiangLiu-github** [comment]
OK, do you know any other references for this application? Thanks for your answer!
